= Differences between Bus Scheme and R5RS

== Parsing

Unsupported:
* Quasiquote, unquote, unquote-splice
* Backslash in strings
* Alternate-base numbers
* Alternate define lambda forms

* []{}| should be reserved

== Functions

=== Basic
Unsupported:
* cond
* case
* let*
* letrec
* do
* delay
* read - uses Ruby's gets

== General
Tail-recursion

Continuations

